recipes/fasttsne/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fastTSNE" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c06115ff228bebe1ff8ef1313baa9e6edda81552c96ab9045f1e4d784852d543
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy >1.14
+    - pip
+    - python >=3.5
+  run:
+    - numba >=0.38.1
+    - numpy >1.14
+    - python >=3.5
+    - scikit-learn >=0.19,<0.19.99
+    - scipy
```

Comment:
Do you need `pynndescent`?

Reply:
Yes, but I couldn't find it on conda-forge, so I copied over the source over from that into a separate [branch](https://github.com/pavlin-policar/fastTSNE/tree/conda-recipe), which I'll merge once I get this working. I've also created a release 0.2.6 which comes from this branch, and the build goes through just fine.